ROS/osr/src/roboclaw_wrapper.py

```diff
@@ -54,17 +55,23 @@ def __init__(self):
 
     def run(self):
         """Blocking loop which runs after initialization has completed"""
-        rate = rospy.Rate(5)
-        mutex_rate = rospy.Rate(10)
+        rate = rospy.Rate(50)
```

Comment:
This is too fast I think. When I run this on my RPi4, it hits ~28Hz. I bet that's going to be less for a 3B(+). Since the controller runs an internal update loop, 10Hz is more than enough. We also have to leave some compute for other purposes, so no need to waste it when it's not necessary.

Reply:
We'll take a look tomorrow and see what the frequency it runs on ours, even at that frequency I didn't think it took that much more computation.